rc/extra/tmux-repl.kak

```diff
@@ -22,7 +24,7 @@ hook global KakBegin .* %sh{
 }
 
 define-command -hidden -params 1..2 tmux-repl-impl %{
-    evaluate-commands %sh{
+    set-option global repl_pane_id %sh{
```

Comment:
In `$TMUX` is not defined, it results in 
```
set-option global repl_pane_id echo -markup '{Error}This command is only available in a tmux session
```
which would fail.